can_bus/quadruna/VC/VC_tx.json

```diff
@@ -84,6 +84,27 @@
             }
         }
     },
+    "INVL_ReadWriteParamCommand": {
+        "msg_id": 33, 
+        "cycle_time": null,
+        "signals": {
+            "INVL_CommandParameterAddress": {
+                "start_bit": 0, 
+                "bits": 16,
+                "start_value": 20
+            },
+            "INVL_CommandReadWrite": {
+                "start_bit": 16, 
+                "enum": "InverterReadWriteCommand",
+                "start_value": 1
+            },
+            "INVL_CommandData": {
```

Comment:
this is based on PM100DZ Can Protocol Manual
https://app.box.com/s/vf9259qlaadhzxqiqrt5cco8xpsn84hk/file/27334613044
(CAN ID is 0x0C1, but now is 33 due to offset)